spec/ParseQuery.spec.js

```diff
@@ -4318,6 +4318,40 @@ describe('Parse.Query testing', () => {
     }
   });
 
+  it_only_db('mongo')('deeply nested Pointers (issue #7413)', async function (done) {
```

Comment:
Is this only a bug in MongoDB, or is this a feature that only works in MongoDB? If the first, then the test should not be exclusively for MongoDB.

Reply:
I don't really know how PostgreSQL works with parse-server, but I think it must be MongoDB only. How sub-documents work in an SQL db ?